cluster/gce/ubuntu/master-helper.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors.
```

Comment:
Changed in https://github.com/kubernetes/kubernetes/pull/44629/commits/f07669ba6c99c02b519addf98459c51f5343d18a